logprep/filter/expression/filter_expression.py

```diff
@@ -120,20 +102,20 @@ class Not(FilterExpression):
     """Filter expression that negates a match."""
 
     def __init__(self, expression: FilterExpression):
-        self.expression = expression
+        self.child = expression
 
     def __repr__(self) -> str:
-        return f"NOT ({str(self.expression)})"
+        return f"NOT ({str(self.child)})"
```

Comment:
it is somehow suspicious to call the `str` method for generation of the `__repr__` dunder method.
```suggestion
        return f"NOT ({repr(self.child)})"
```

Reply:
I've changed it.